src/theme/ui_styles.rs

```diff
@@ -192,80 +194,82 @@ impl UiStyles {
+
+            "ga" => self.git.new                                    = pair.to_style(),
+            "gm" => self.git.modified                               = pair.to_style(),
+            "gd" => self.git.deleted                                = pair.to_style(),
+            "gv" => self.git.renamed                                = pair.to_style(),
+            "gt" => self.git.typechange                             = pair.to_style(),
+            "gi" => self.git.ignored                                = pair.to_style(),
+            "gc" => self.git.conflicted                             = pair.to_style(),
+
+            "xx" => self.punctuation                                = pair.to_style(),
+            "da" => self.date                                       = pair.to_style(),
+            "in" => self.inode                                      = pair.to_style(),
+            "bl" => self.blocks                                     = pair.to_style(),
+            "hd" => self.header                                     = pair.to_style(),
+            "oc" => self.octal                                      = pair.to_style(),
+            "lp" => self.symlink_path                               = pair.to_style(),
+            "cc" => self.control_char                               = pair.to_style(),
+            "bO" => self.broken_path_overlay                        = pair.to_style(),
+
+            "mp" => self.filekinds.mount_point                      = pair.to_style(),
+            "sp" => self.filekinds.special                          = pair.to_style(),  // Catch-all for unrecognized file kind
+
+            "im" => self.file_type.image                            = pair.to_style(),
+            "vi" => self.file_type.video                            = pair.to_style(),
+            "mu" => self.file_type.music                            = pair.to_style(),
+            "lo" => self.file_type.lossless                         = pair.to_style(),
+            "cr" => self.file_type.crypto                           = pair.to_style(),
+            "do" => self.file_type.document                         = pair.to_style(),
+            "co" => self.file_type.compressed                       = pair.to_style(),
+            "tm" => self.file_type.temp                             = pair.to_style(),
+            "cm" => self.file_type.compiled                         = pair.to_style(),
+            "bu" => self.file_type.build                            = pair.to_style(),
+
+            "Sn" => self.security_context.none                      = pair.to_style(),
+            "Su" => self.security_context.selinux.user              = pair.to_style(),
+            "Sr" => self.security_context.selinux.role              = pair.to_style(),
+            "St" => self.security_context.selinux.typ               = pair.to_style(),
+            "Sl" => self.security_context.selinux.range             = pair.to_style(),
```

Comment:
Diffs like this makes it hard to see what's changed, could you, if you wanna do formatting changes, perhaps do them in a separate `refactor(<filename>): reformat` commit?